var/spack/repos/builtin/packages/py-easybuild-easyblocks/package.py

```diff
@@ -12,8 +12,14 @@ class PyEasybuildEasyblocks(PythonPackage):
     """
 
     homepage = 'http://hpcugent.github.io/easybuild/'
-    url      = 'https://pypi.io/packages/source/e/easybuild-easyblocks/easybuild-easyblocks-3.1.2.tar.gz'
+    url      = 'https://pypi.io/packages/source/e/easybuild-easyblocks/easybuild-easyblocks-4.0.0.tar.gz'
+    maintainers = ['boegel']
 
+    version('4.0.0', sha256='a0fdef6c33c786e323bde1b28bab942fd8e535c26842877d705e692e85b31b07')
     version('3.1.2', 'be08da30c07e67ed3e136e8d38905fbc')
 
-    depends_on('py-easybuild-framework@3.1:', when='@3.1:', type='run')
+    depends_on('python@2.6:2.8', when='@:3', type=('build', 'run'))
+    depends_on('python@2.6:2.8,3.5:', when='@4:', type=('build', 'run'))
+
+    for v in ['@3.1:', '@4.0:']:
```

Comment:
This is not correct, since `easybuild-easyblocks` 4.0.1 is likely to require `easybuild-framework` 4.0.1 (same for 4.1.0, etc.)